elements/pfe-primary-detail/src/pfe-primary-detail--lightdom.scss

```diff
@@ -0,0 +1,24 @@
+@import "../../pfe-sass/pfe-sass";
```

Comment:
I think this should probably be in the component's Sass? Is it in a light DOM stylesheet because it's slotted?

You can follow this example for how to exert styling onto the slotted elements, I'm not sure how `::slotted()` works with pseudo-classes, maybe @castastrophe can advise on that?
https://github.com/patternfly/patternfly-elements/blob/879b4d7e8edb55990dc683512f3b77d0ed4744e0/elements/pfe-primary-detail/src/pfe-primary-detail.scss#L47

Reply:
@wesruv I tried adding the styles in the component sass with `::slotted` selector as you have shown but I could not get it to work with the pseudo classes. Maybe I was just not implementing it properly. I'll wait to hear back from Cassondra.

Reply:
Slotted elements cannot style on pseudo-selectors (a good reference PR on this issue here -> https://github.com/WICG/webcomponents/issues/655 -- which interestingly Kyle linked to in March of last year too!)

- <s>Support other pseudo elements inside ::slotted(): ::slotted(div::before)</s>
- Support other pseudo elements after ::slotted(): ::slotted(div)::before


`the tree-abiding pseudos (::before, ::after, ::marker, and ::placeholder) can all go after ::slotted() now.`

I'm seeing it fixed in Chromium but not seeing any notes about Safari so 🤷 .  Seems like an unstable option either way so I would suggest adding a tag and applying styles to it if you can.

Reply:
Also Safari has some issues with focus-within (see pfe-cta for workaround ideas).  We used JS to add a focus-within class to the wrapper and then we can style based on that.  That's an option for handling focus when you're trying to target light DOM.

Reply:
So maybe as a workaround we add classes when the elements are being hovered/focused on?